mkdocs_static_i18n/suffix.py

```diff
@@ -154,9 +158,8 @@ def on_files(self, files, config):
                     log.debug(f"Ignore {i18n_file.locale} {i18n_file}")
 
         # theme (and overrides) files
-        else:
-            if self.is_default_language_build:
-                i18n_files.append(file)
+        elif self.is_default_language_build or file.src_uri.startswith("assets/"):
```

Comment:
Can you explain why you're adding `or file.src_uri.startswith("assets/")` please? I must admit that I don't understand.

Reply:
Okay, funny to see all those possible use cases :brain: